mono/metadata/marshal-ilgen.c

```diff
@@ -4253,28 +4269,6 @@ emit_marshal_custom_ilgen (EmitMarshalContext *m, int argnum, MonoType *t,
 	mklass = mono_class_from_mono_type (mtype);
 	g_assert (mklass != NULL);
 
-handle_exception:
```

Comment:
Isn't this is a significant semantic change? It was previously falling into the label, or goto'ing it. Now the falling is missed.

Reply:
I see now. This was maybe indentation reduction, or hot/cold split for efficiency, or human wanted exceptional path out of the way for easy reading, or mis-maintenance.